gcs/CHANGES.md

```diff
@@ -21,6 +21,10 @@
 
 1. Increasing gRPC read timeout from 30 seconds to 3600 seconds.
 
+1. Updating max list items per call from 1024 -> 5000. (#872)
```

Comment:
please follow the pattern and change "->" to "to"?

Reply:
addressed.